srcs/Config/ServerConfig/ServerConfig.hpp

```diff
@@ -0,0 +1,82 @@
+#pragma once
+
+#include <algorithm>
+#include <iostream>
+#include <map>
+#include <sstream>
+#include <string>
+#include <vector>
+#include "ConfigHandlingString.hpp"
+#include "HandlingString.hpp"
+#include "NumericHandle.hpp"
+
+class ServerConfig
+{
+	private:
+		bool									_autoindex;
+		bool									_chunked_transferencoding_allow;
+		int										_server_tokens;
+		size_t									_client_body_buffer_size;
+		size_t									_client_body_timeout;
+		size_t									_client_header_buffer_size;
+		size_t									_client_header_timeout;
+		size_t									_keepalive_requests;
+		size_t									_keepalive_timeout;
+		size_t 									_client_max_body_size;
+		std::string								_accesslog;
+		std::string								_cgi_extension;
+		std::string								_default_type;
+		std::string								_errorlog;
+		std::string								_port;
+		std::string								_root;
+		std::vector<std::string>				_allow_methods;
+		std::vector<std::string>				_index;
+		std::vector<std::string>				_server_name;
+		void	init_server_config();
+	public:
+		ServerConfig();
+		ServerConfig(const ServerConfig &other);
+		ServerConfig& operator=(const ServerConfig &other);
+		~ServerConfig();
+
+		// getter
+		bool									get_autoindex() const;
+		bool									get_chunked_transferencoding_allow() const;
+		int										get_server_tokens() const;
+		size_t									get_client_body_buffer_size() const;
+		size_t									get_client_body_timeout() const;
+		size_t									get_client_header_buffer_size() const;
+		size_t									get_client_header_timeout() const;
+		size_t									get_keepalive_requests() const;
+		size_t									get_keepalive_timeout() const;
+		size_t 									get_client_max_body_size() const;
+		std::string								get_accesslog() const;
+		std::string								get_default_type() const;
```

Comment:
```suggestion
		std::string get_cgi_extension() const;
		std::string								get_default_type() const;
```

`cgi_extension`のgetterを追加しました。
`TestConfigReading.cpp`で漏れなく項目を評価していれば、getterの欠損に気付けそうでした。